### ISSUE TITLE
About the protein sequences of pool a and pool b

### ISSUE THREAD

**Issue by wuwo007** [statisticalbiotechnology/proteoform-standard#1]
Thank for you sharing this standard dataset for protein inference. I have the associated publication "The,M. et al. (2018) A Protein Standard That Emulates Homology for the Characterization of Protein Inference Algorithms. J. Proteome Res., 17, 1879–1886.". However i have some confusion about the protein sequences of pool a and pool b:
1, Why the number of protein sequences in "prest_pool_a.fasta" is 192? According to the description in the above paper "To generate the datasets, the PrEST sequences of the Human Proteome Atlas-project were scanned for a total of 191 overlapping pairs of PrEST sequences. The resulting protein pairs share at least one identical fully tryptic peptide. Each pair originates from different human genes and flanking amino acids around the tryptic termini is thereby not shared between them. Each recombinant protein was randomly assigned into one of the pools (A or B) that were created ", should't the number of proteins of pool a or pool b be 191?
2, I compare the protein sequences of pool a and pool b by the website tool BLAST (https://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE=Proteins&PROGRAM=blastp&PAGE_TYPE=BlastSearch&BLAST_SPEC=blast2seq ) and find that : 
1) In pool a, there are 2 proteins which do not have overlapping sequences with pool b's proteins:
HPRR650006 and HPRR1320009, the following is the BLAST results:
Query= HPRR650006
Length=147                                                                 Score     E
Sequences producing significant alignments:                       (Bits)  Value
Query_185747 HPRR3790495                                           17.3    5.1  
Query_185796 HPRR4050261                                           17.3    5.7  
ALIGNMENTS
>Query_185747 HPRR3790495
Length=58
 Score = 17.3 bits (33),  Expect = 5.1, Method: Compositional matrix adjust.
 Identities = 5/8 (63%), Positives = 6/8 (75%), Gaps = 0/8 (0%)
Query  104  HFILVECC  111
            HF+L E C
Sbjct  50   HFLLCEAC  57
>Query_185796 HPRR4050261
Length=74
 Score = 17.3 bits (33),  Expect = 5.7, Method: Compositional matrix adjust.
 Identities = 6/20 (30%), Positives = 10/20 (50%), Gaps = 0/20 (0%)
Query  90   FQYIRNCVMNRALPHFILVE  109
            F +   C +N+   H IL +
Sbjct  13   FGFRLGCALNQNGVHVILFD  32

Query= HPRR1320009
Length=92                                                                 Score     E
Sequences producing significant alignments:                       (Bits)  Value
Query_185827 HPRR4200118                                           20.8    0.12 
Query_185766 HPRR3940080                                           19.6    0.44 
Query_185773 HPRR3970058                                           17.3    2.8  
Query_185811 HPRR4120251                                           17.3    3.5  

ALIGNMENTS
>Query_185827 HPRR4200118
Length=50
 Score = 20.8 bits (42),  Expect = 0.12, Method: Compositional matrix adjust.
 Identities = 10/18 (56%), Positives = 12/18 (67%), Gaps = 1/18 (6%)
Query  48  RLDRNLWGLAGLNSCPVK  65
           R+D N +GLAGL   P K
Sbjct  16  RMDSN-FGLAGLPGSPKK  32
>Query_185766 HPRR3940080
Length=62
 Score = 19.6 bits (39),  Expect = 0.44, Method: Compositional matrix adjust.
 Identities = 11/28 (39%), Positives = 16/28 (57%), Gaps = 0/28 (0%)
Query  41  QLIRFLKRLDRNLWGLAGLNSCPVKEAN  68
           QL RFL+  D+    +  +NS  V EA+
Sbjct  8   QLQRFLENQDKLFRQVTFVNSKTVTEAS  35
>Query_185773 HPRR3970058
Length=73
 Score = 17.3 bits (33),  Expect = 2.8, Method: Compositional matrix adjust.
 Identities = 8/19 (42%), Positives = 14/19 (74%), Gaps = 1/19 (5%)
Query  45  FLKRLDRNL-WGLAGLNSC  62
           + KRLDR++ + L+ L+S 
Sbjct  36  YSKRLDRSVSYQLSPLDST  54
>Query_185811 HPRR4120251
Length=83
 Score = 17.3 bits (33),  Expect = 3.5, Method: Compositional matrix adjust.
 Identities = 5/9 (56%), Positives = 7/9 (78%), Gaps = 0/9 (0%)
Query  47  KRLDRNLWG  55
           + L +NLWG
Sbjct  44  RNLGQNLWG  52

2) In the pool b, there are 7 proteins which do not have overlapping sequences with the pool a's proteins:
HPRR3690184
HPRR3700338
HPRR3730410
HPRR3870348
HPRR4030177
HPRR4350096
HPRR4470317
The BLAST results of these proteins is omited due to the limited space.

Could you help me to explain these results?

**Comment by percolator** [comment]
Wuwo, we are looking into it.

**Comment by wuwo007** [comment]
Thanks. Wait for your reply.

**Comment by percolator** [comment]
Hi,

It seems like you are right. The text in the manuscript,
unfortunately, reflects an intention rather than the reality. For the
examples you mention, there are no shared peptides between the pools. We
had looked up relevant constructs (PrESTs) containing shared peptides, but
some of those constructs were out of stock. We never got that right when
writing up the manuscript.

Hope this clarifies what you are seeing in the fasta-files.

Cheers
--Lukas

**Comment by wuwo007** [comment]
Dear Lukas,


Thank you for the reply. I will take it as a wrong statement in your manuscript. In my opinion, this will not effect the dataset's designed goal for benchmarking proteoforms inference algorithms. Recently, I am developing a new protein inference algorithm and would like to use this elaborate dataset. However, I still have a question when reading your manuscript "A Protein Standard That Emulates Homology for the Characterization of Protein Inference Algorithms (2018 JPR)". Could you please tell me that what is the differences between entrapment FDR and Decoy FDR in your manuscript? And how to calculate them?


Looking forward for your reply.


Gao
|
|
|
|
|
On 11/13/2018 23:10，Lukas Käll<notifications@github.com> wrote：
Hi,

It seems like you are right. The text in the manuscript,
unfortunately, reflects an intention rather than the reality. For the
examples you mention, there are no shared peptides between the pools. We
had looked up relevant constructs (PrESTs) containing shared peptides, but
some of those constructs were out of stock. We never got that right when
writing up the manuscript.

Hope this clarifies what you are seeing in the fasta-files.

Cheers
--Lukas

**Comment by percolator** [comment]
Hi Gao,

I will try to come up with a way to make the distinction visible to the
users of the standard. It will not be straightforward to do so, as the
previous incorrect statement is easier to convey than the more complex
reality.

As for entrapment databases, here is an article that describes them in
detail:
https://pubs.acs.org/doi/abs/10.1021/pr1012619

In short, the entrapment database is a collection of sequences known not to
be present in the sample of known content. For the paper, we included a
script which will calculate entrapment FDR for you:
https://github.com/statisticalbiotechnology/proteoform-standard/blob/master/generate_prest_report.py

Yours
--Lukas

**Comment by wuwo007** [comment]
Dear Lukas,


Thanks for your kind reply. 


Yours
Gao
On 11/14/2018 15:53，Lukas Käll<notifications@github.com> wrote：
Hi Gao,

I will try to come up with a way to make the distinction visible to the
users of the standard. It will not be straightforward to do so, as the
previous incorrect statement is easier to convey than the more complex
reality.

As for entrapment databases, here is an article that describes them in
detail:
https://pubs.acs.org/doi/abs/10.1021/pr1012619

In short, the entrapment database is a collection of sequences known not to
be present in the sample of known content. For the paper, we included a
script which will calculate entrapment FDR for you:
https://github.com/statisticalbiotechnology/proteoform-standard/blob/master/generate_prest_report.py

Yours
--Lukas
